main.py

```diff
@@ -1,10 +1,41 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_set = json.load(f)
+    print(players_set)
+
+    for user, info_user in players_set.items():
+
+        race, _ = Race.objects.get_or_create(
+            name=info_user["race"]["name"],
+            description=info_user["race"]["description"]
+        )
```

Comment:
Unpacking Issue: Ensure to unpack the results of `get_or_create` for the `Race` model to handle the object and the `created` flag correctly.